pkgs/development/ocaml-modules/tsdl/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchgit, ocaml, findlib, ctypes, result, SDL2, pkgconfig, opam }:
+
+let
+  inherit (stdenv.lib) getVersion;
+
+  pname = "tsdl";
+  version = "0.9.0";
+
+in
+
+stdenv.mkDerivation {
+  name = "ocaml-${pname}-${version}";
+
+  src = fetchgit {
+    url = "http://erratique.ch/repos/tsdl.git";
+    rev = "refs/tags/v${version}";
+    sha256 = "0px6mksc9d8jlkjh7kj1g1jsak21vam2wpdna6xs8d6ihii8frnf";
+  };
+
+  buildInputs = [ ocaml findlib ctypes result pkgconfig opam ];
+  propagatedBuildInputs = [ SDL2 ];
+
```

Comment:
@vbgl You are right, I am getting the same error as well... I had previously added `ctypes` to my `buildInputs` but later forgotten about it.

I will fix everything you said. Thanks!